src/components/Checkbox/index.js

```diff
@@ -0,0 +1,55 @@
+/** @jsx jsx */
+import React from 'react'
+import { jsx } from '@emotion/core'
+import Text from 'components/Text'
+import { TEXT_TYPE } from 'components/Text/constants'
+import { ReactComponent as Check } from 'static/icons/check.svg'
+
+import styles from './Checkbox.styles'
+
+class InputCheckbox extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      checked: false,
+    }
+  }
+
+  onChange = () => {
+    if (this.props.customOnChange) {
+      this.props.customOnChange(!this.state.checked)
+    }
+    this.setState({ checked: !this.state.checked })
+  }
+
+  render() {
+    const { label, customOnChange, ...rest } = this.props
+
+    return (
+      <label css={styles.label}>
+        <input
+          checked={this.state.checked}
+          css={styles.input}
+          type="checkbox"
+          onChange={this.onChange}
+          {...rest}
```

Comment:
This can be pretty error prone. Is it possible to get these values one by one without too much pain?